src/app/models/question.model.js

```diff
@@ -4,6 +4,7 @@ let questionSchema = mongoose.Schema({
   text : { type: String, required: true, maxlength: 1000 },
   user : { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true },
   answered: Boolean,
```

Comment:
Right, it's not really necessary, but I left it to support older questions. Don't know if problems will arise if I remove it.